woo-razorpay.php

```diff
@@ -2150,6 +2150,10 @@ public function update1ccOrderWC(& $order, $wcOrderId, $razorpayPaymentId)
                 if($razorpayData['shipping_fee'] == 0)
                 {
                     $item->set_method_title( 'Free Shipping' );
+                    if(isset($razorpayData['notes']) && isset($razorpayData['notes']['shipping_method_name']))
```

Comment:
`notes` key will be always present so I don't think we need to check `isset($razorpayData['notes'])`